lang/cs/build.props

```diff
@@ -133,4 +134,10 @@ under the License.
       DestinationFiles="@(MySourceFiles->'$(TargetDir)%(Filename)%(Extension)')"
     />
   </Target>
+    <ItemGroup>
+        <DirectoriesToRemove Include="$(IntermediateOutputPath)" />
+     </ItemGroup>
+     <Target Name="RemoveIntermediateDirectories"  AfterTargets="clean">
+        <RemoveDir Directories="@(DirectoriesToRemove)" Condition="!Exists('$(IntermediateOutputPath)\$(RootNamespace).log')"/>
```

Comment:
It cleans temp folder. Not dlls. Shall we define Clean target to clean $(OutputPath)?